### DISCUSSION TITLE
sha256 automatic urls

### DISCUSSION THREAD

**Discussion by AndreasHart** [vercel/next.js#38250] [Help]
how do you create the sha256 on Github deploys through api?

echo "git-$($BRANCH_NAME)-web-app" | sha256sum | cut -c1-6"

is the bash command to get the hash I have been using
I've tried several but have yet to get it right.

thanks!

https://vercel.com/docs/concepts/deployments/automatic-urls#truncation

**Comment by mohitshri061** [comment]
e9f37061f36bef8920768a0a8b9475e96d80c3cb3c7d43a8b471955592c1a745
